SQL/0000-00-01-Permission.sql

```diff
@@ -101,7 +101,8 @@ INSERT INTO `permissions` VALUES
     (52,'instrument_manager_write', 'Instrument Manager: Install new instruments via file upload', 2),
     (53,'publication_view', 'Publication - Access to module', 2),
     (54,'publication_propose', 'Publication - Propose a project', 2),
-    (55,'publication_approve', 'Publication - Approve or reject proposed publication projects', 2);
+    (55,'publication_approve', 'Publication - Approve or reject proposed publication projects', 2),
+    (56,'document_repository_edit','Upload and edit files in Document Repository','2');
```

Comment:
Are you proposing 3 doc repo permissions? Is this new `document_repository_edit ` permission also separate from the `document_repository_edit ` permission?

Reply:
I just wanted to separate view from edit, basically. 

I'm not sure whether there should also be a delete. Maybe there is a use case where someone is allowed to edit information about doc repo files but shouldn't be able to remove them entirely?

Reply:
I agree with separating view and upload / edit. 

I'm not sure we need that much granularity in terms of edit / delete, but I'm not strictly opposed to it